python_toolbox/evaluation/plot.py

```diff
@@ -36,53 +36,80 @@
 
-	label_str = 'recall'
-	ax.plot(edges_target[1::], cum_target*100, c='blue',
-			label=label_str, linewidth=2.0)
+    ax = plt.subplot(111)
+    label_str = "precision"
+    ax.plot(
+        edges_source[1::],
+        cum_source * 100,
+        c="red",
+        label=label_str,
+        linewidth=2.0,
+    )
 
-	ax.grid(True)
-	plt.rcParams["figure.figsize"] = plt_size
-	plt.rc('axes', prop_cycle=cycler('color', ['r', 'g', 'b', 'y']))
-	plt.title('Precision and Recall: ' + scene + ', ' + "%02.2f f-score"%(fscore*100))
-	plt.axvline(x = dist_threshold, c='black', ls='dashed', linewidth=2.0)
+    label_str = "recall"
+    ax.plot(
+        edges_target[1::],
+        cum_target * 100,
+        c="blue",
+        label=label_str,
+        linewidth=2.0,
+    )
 
-	plt.ylabel('# of points (%)', fontsize=15)
-	plt.xlabel('Meters', fontsize=15)
-	plt.axis([0, dist_threshold*plot_stretch, 0, 100])
-	ax.legend(shadow=True, fancybox=True, fontsize=pfontsize)
-	#plt.axis([0, dist_threshold*plot_stretch, 0, 100])
+    ax.grid(True)
+    plt.rcParams["figure.figsize"] = plt_size
+    plt.rc("axes", prop_cycle=cycler("color", ["r", "g", "b", "y"]))
+    plt.title(
+        "Precision and Recall: "
+        + scene
+        + ", "
+        + "%02.2f f-score" % (fscore * 100)
+    )
+    plt.axvline(x=dist_threshold, c="black", ls="dashed", linewidth=2.0)
 
-	plt.setp(ax.get_legend().get_texts(), fontsize=pfontsize)
+    plt.ylabel("# of points (%)", fontsize=15)
+    plt.xlabel("Meters", fontsize=15)
+    plt.axis([0, dist_threshold * plot_stretch, 0, 100])
+    ax.legend(shadow=True, fancybox=True, fontsize=pfontsize)
+    # plt.axis([0, dist_threshold*plot_stretch, 0, 100])
 
-	plt.legend(loc=2, borderaxespad=0., fontsize=pfontsize)
-	plt.legend(loc=4)
-	leg = plt.legend( loc = 'lower right')
+    plt.setp(ax.get_legend().get_texts(), fontsize=pfontsize)
 
-	box = ax.get_position()
-	ax.set_position([box.x0, box.y0, box.width * 0.8, box.height])
+    plt.legend(loc=2, borderaxespad=0.0, fontsize=pfontsize)
+    plt.legend(loc=4)
+    leg = plt.legend(loc="lower right")
 
-	# Put a legend to the right of the current axis
-	ax.legend(loc='center left', bbox_to_anchor=(1, 0.5))
-	plt.setp(ax.get_legend().get_texts(), fontsize=pfontsize)
-	png_name = mvs_outpath+ "/PR_{0}_@d_th_0_{1}.png".format(
-			scene,"%04d"%(dist_threshold*10000))
-	pdf_name = mvs_outpath+ "/PR_{0}_@d_th_0_{1}.pdf".format(
-			scene,"%04d"%(dist_threshold*10000))
+    box = ax.get_position()
+    ax.set_position([box.x0, box.y0, box.width * 0.8, box.height])
 
-	# save figure and display
-	f.savefig(png_name, format='png', bbox_inches='tight')
-	f.savefig(pdf_name, format='pdf', bbox_inches='tight')
-	if show_figure:
-		plt.show()
+    # Put a legend to the right of the current axis
+    ax.legend(loc="center left", bbox_to_anchor=(1, 0.5))
+    plt.setp(ax.get_legend().get_texts(), fontsize=pfontsize)
+    png_name = mvs_outpath + "/PR_{0}_@d_th_0_{1}.png".format(
```

Comment:
I see some Python3 scripts (I am not trying to blame this PR, but I just want to mention). How about saying Python3 in the requirement or readme?

Reply:
Pyhton3 as a requirement is good with me.

Reply:
I added it as req to the readme